wolfcrypt/src/poly1305.c

```diff
@@ -778,7 +778,7 @@ int wc_Poly1305Update(Poly1305* ctx, const byte* m, word32 bytes)
 
         /* process full blocks */
         if (bytes >= POLY1305_BLOCK_SIZE) {
-            size_t want = ((size_t)bytes & ~(POLY1305_BLOCK_SIZE - 1));
+            size_t want = ((size_t)bytes & ~((size_t)POLY1305_BLOCK_SIZE - 1));
```

Comment:
Change this to POLY1305_BLOCK_MASK.

Reply:
:heavy_check_mark:

Reply:
There isn't a `POLY1305_BLOCK_MASK` defined anywhere -- were you thinking of adding it?  Seems out of scope for the PR.

OTOH -- gack I just realized we don't even have a macro for static asserts -- I'd be more comfortable if these sorts of constructs, which assume a constant is a power of 2, had static asserts to make sure, à la `#define WOLF_STATIC_ASSERT_POWER_OF_2(x) static_assert(x & (x-1) == 0, "expected " #x " to be a power of 2")`.

Also out of scope...